inputs/prod_customize.json

```diff
@@ -0,0 +1,7 @@
+{
+    "disable_plugins": [
+    ],
+
+    "enable_plugins": [
+    ]
+}
```

Comment:
To be clear: this is an example customize file and will not be shipped in the package/RPM, is that correct? That way makes sense to me.